src/components/registration/steps.jsx

```diff
@@ -27,6 +27,17 @@ var Tooltip = require('../../components/tooltip/tooltip.jsx');
 require('./steps.scss');
 
 var DEFAULT_COUNTRY = 'us';
+var COUNTRY_OPTIONS = countryData.countryOptions.concat({
+    label: <intl.FormattedMessage id="teacherRegistration.selectCountry" />,
+    disabled: true,
+    selected: true
+}).sort(function (a, b) {
+    if (a.disabled) return -1;
+    if (b.disabled) return 1;
+    if (a.value === DEFAULT_COUNTRY) return -1;
```

Comment:
I didn't realize that's how it was working before – I didn't see that `teacherregistration` was doing that. Why don't we just have the value selected be the country from before, rather than sorting it to the top?